src/pages/invoices/index/Invoices.tsx

```diff
@@ -12,39 +12,28 @@ import { useTitle } from 'common/hooks/useTitle';
 import { DataTable } from 'components/DataTable';
 import { Default } from 'components/layouts/Default';
 import { useTranslation } from 'react-i18next';
-import { Link as ReactRouterLink } from 'react-router-dom';
-import { Download } from 'react-feather';
 import { useActions } from '../edit/components/Actions';
 import {
   defaultColumns,
   invoiceColumns,
   useInvoiceColumns,
 } from '../common/hooks/useInvoiceColumns';
 import { DataTableColumnsPicker } from 'components/DataTableColumnsPicker';
+import { useInvoiceFilters } from '../common/hooks/useInvoiceFilters';
+import { ImportButton } from 'components/import/ImportButton';
 
 export function Invoices() {
   const { documentTitle } = useTitle('invoices');
-  const { t } = useTranslation();
 
-  const pages = [{ name: t('invoices'), href: '/invoices' }];
+  const [t] = useTranslation();
 
-  const importButton = (
-    <ReactRouterLink to="/invoices/import">
-      <button className="inline-flex items-center justify-center py-2 px-4 rounded text-sm text-white bg-green-500 hover:bg-green-600">
-        <svg
-          className="w-4 h-4 mr-2"
-          xmlns="http://www.w3.org/2000/svg"
-          viewBox="3 3 20 20"
-        >
-          <Download />
-        </svg>
-        <span>{t('import')}</span>
-      </button>
-    </ReactRouterLink>
-  );
```

Comment:
I don't know how I feel about this. I think it's overengineering for such a simple button to have the additional file. Do we need to reuse this button? If not, please keep it as is.

Also, I noticed you used the same pattern on the transactions page, so please make sure this is aligned with the rest of the project.

Reply:
I agree with you that this is a very simple button, but if we're just changing the route every time I think that's a good reason to globalize it. Also, if we include using the same button in 5 or more places, I think it's good for globalization. I implemented this component in every place where I had some functions or refactoring. So I'll go ahead with that, or if you want, maybe I can do a separate PR to implement this button everywhere.